src/main/java/org/sunbird/operationalreports/service/OperationalReportServiceImpl.java

```diff
@@ -367,6 +372,30 @@ private long getParsedDate(String reportExpiryDate) throws ParseException {
         return reportExpiryDateMillis;
     }
 
+
+    public SBApiResponse getFileInfo(String authToken) {
```

Comment:
should have try catch and from catch block should update response object with 500 error details

Reply:
have created a try catch block and returned the internal server error response in the catch block